cpp/velox/compute/VeloxPlanConverter.cc

```diff
@@ -587,6 +598,25 @@ class VeloxPlanConverter::WholeStageResIterFirstStage
   std::vector<core::PlanNodeId> streamIds_;
   std::vector<std::vector<exec::Split>> splits_;
   bool noMoreSplits_ = false;
+
+  // Extract the partition column and value from a path of split.
+  // Extraction for multiple partition columns is not supported.
+  std::pair<std::string, std::string> extractPartitionColumnAndValue(
+      const std::string& filePath) {
+    std::pair<std::string, std::string> partitionColumnAndValue("", "");
+    std::string delimiter = "=";
```

Comment:
sometimes `=` can be in column name, but this is a corner case. we do not need to consider now

Reply:
Added a comment for this case.